### ISSUE TITLE
Direwolf20 server crashing on startup. please help. :(

### ISSUE THREAD

**Issue by noslo012** [Chicken-Bones/WirelessRedstone#20]
I haven't used this site before, so I apologize in advance if i am posting in the wrong place or something. Anyway, it was recommended to me that I post here. I have recently done a clean install of everything involving FTB, including the server I've been hosting for the last two months. Now that I've reinstalled it all, my server just wants to crash when I launch it. I'm sure it's something very simple, however, I don't know how to read/interpret logs. But I DO know how to post them, haha. I am hoping that someone can look at this for me and tell me what's happening and/or how to fix it. I posted on Reddit and someone told me that it looks like a bug with WR-CBE.

Thanks a million!

**Comment by noslo012** [comment]
Just realized I forgot to include a link to the log. Oops. Here it is.
http://paste.feed-the-beast.com/view/1cab9578

**Comment by Rushmead** [comment]
Just a thought, have you tried removing Biomes o Plenty?

**Comment by noslo012** [comment]
No, I have not tried that. The world file is already created. I have been hosting it fine without any problems for several weeks now. Long story, but I ended up needing to reinstall everything. Now that I've done so, nothing is working in my favor.

Could i still remove BoP even if the world already exists? Sounds like it would create a whole mess of other problems.

**Comment by Rushmead** [comment]
Ok, what version are you running?

**Comment by noslo012** [comment]
Direwolf20 1.7.10 is the version of the modpack that our world was created with a few weeks ago.
As for the server, that's the part I can't figure out. I've tried the "Recommended" button at the top of the FTB Launcher, as well as the 1.9.0 option, the 1.8.0 option, and the 1.7.0 option. All of which have given me the same problem that I posted above.

When I click the ServerStart.bat file, it sorta does its thing as it tries to load (you know, with all the white text in the cmd window), and when it's finished, it says at the bottom, 
"[FML] The state engine was in incorrect state SERVER_STARTING and forced into state SERVER_STOPPED. Errors may have been discarded.
Server process finished
Press any key to continue . . ."

Does the same thing on all of the different server versions I've tried so far.

**Comment by noslo012** [comment]
Ohh, hey I just had an idea of what could be going on. I took notice that in the FTB Launcher, the Direwolf20 1.7.10 pack that I had used before now says it's the 1.8.0 pack (I'm guessing it was changed in some recent update). So I guess that means that it's trying to download the server for 1.8.0. Where can i download the server for the 1.7.0 of the modpack?

**Comment by Rushmead** [comment]
So you want to run the MC 1.7.10 direwolf20 1.7.10 version? All downloads are available here . http://www.feed-the-beast.com/projects/ftb-presents-direwolf20/files Most of them have additional files at the bottom of the page AKA the server downloads

**Comment by noslo012** [comment]
Are these the server downloads? Or are they just for the game itself?

**Comment by Rushmead** [comment]
As i said. Server downloads are available via the "+1 more" on the files

**Comment by noslo012** [comment]
Oooh, this looks promising, I will try this and see what happens. Unfortunately, I have to leave for work here in a few minutes, so if my response is late, I apologize. 

FINGERS CROSSED!

**Comment by noslo012** [comment]
Okay, I am home now and back-at-it again. It pains me to report that downloading the 1.7.10 server didn't work. I am completely stumped here. :(
This really bites.

**Comment by noslo012** [comment]
I FIGURED IT OUT! After three days of fooling with this NONSENSE, I've finally got this working again.
Turns out, my world was corrupted somehow? Don't know how that happened.
I followed this guide and it worked. What a miracle.
http://minecraft.gamepedia.com/Tutorials/Recover_corrupted_saved_world_data
Thank you for helping me
